manager/integration/tests/test_node.py

```diff
@@ -1002,6 +1002,30 @@ def test_node_controller_sync_disk_state(client):  # NOQA
                                      CONDITION_STATUS_TRUE)
 
 
+@pytest.mark.skip(reason="TODO")
+@pytest.mark.node  # NOQA
+@pytest.mark.mountdisk # NOQA
+def test_node_default_disk_addedback_with_extra_disk_unmounted(client):  # NOQA
+    """
+    [Node] Test adding default disk back with extra disk is unmounted
+    on the node
+
+    1. Create and attach a new (Linux) disk to node 1.
+    2. Create an schedulable extra Longhorn disk on node 1 with
```

Comment:
The first two steps can be simple as:
> 1. Create a host disk `extra_disk` using Longhorn volume and attach it to node 1.

`host disk` refers to Longhorn's disk created using Longhorn volume in our test cases.